controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -395,20 +391,21 @@ func (r *StorageConsumerReconciler) reconcileCephClientRBDProvisioner() error {
 		return nil
 	})
 
-	cephClientRBDProvisionerDetails := &ocsv1alpha1.CephResourcesSpec{
-		Kind: r.cephClientRBDProvisioner.Kind,
-		Name: r.cephClientRBDProvisioner.Name,
+	if err != nil {
+		r.Log.Error(err,
+			"Failed to update CephClient.",
+			"CephClient",
+			klog.KRef(r.cephClientRBDProvisioner.Namespace, r.cephClientRBDProvisioner.Name),
+		)
+		return err
 	}
+
+	var phase string
```

Comment:
To make it more obvious that we are willing to intentionally use an empty string
```suggestion
phase := "" 
```